### DISCUSSION TITLE
Welcome to Anima Discussions!

### DISCUSSION THREAD

**Discussion by lyogavin** [lyogavin/airllm#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Deh-alba** [comment]
Congrats for the iniciative!!!
Hey, the library works with just llama2.0?
How i can do download models with login?

**Reply by lyogavin** [reply]
We can add a token parameter. will do that.

**Reply by Deh-alba** [reply]
would be nice, will help a lot

**Reply by lyogavin** [reply]
added in airllm==2.4.2. Please give it a try.

**Comment by ZorXL** [comment]
airllm is just amazing. I am not sure how big work is necessary to add a simplest UI and make it an exe file, for Windows users.. It would be really  attractive .
